src/calculateRentalCost.js

```diff
@@ -4,7 +4,9 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  console.log('No solution at all');
```

Comment:
Remove the `console.log` statement as it is not part of the final solution and should not be present in production code.